libs/ledger/src/chain/block_coordinator.cpp

```diff
@@ -109,6 +110,38 @@ BlockCoordinator::BlockCoordinator(MainChain &chain, DAGPtr dag, StakeManagerPtr
   , exec_wait_periodic_{EXEC_NOTIFY_INTERVAL}
   , syncing_periodic_{NOTIFY_INTERVAL}
   , synergetic_exec_mgr_{CreateSynergeticExecutor(features, dag, storage_unit_)}
+  , reload_state_count_{telemetry::Registry::Instance().CreateCounter(
```

Comment:
Counter names should probably be dot-delimited; that's normal for tools like Graphite.

Reply:
Looks like that is not the case in prometheus. I want to get these changes in so that we can start experimenting with the client QL.